integration/operator_test.go

```diff
@@ -252,27 +252,31 @@ func operatorSetup() error {
 		return microerror.Mask(err)
 	}
 
-	if err := writeAWSResourceValues(); err != nil {
-		return microerror.Maskf(err, "unexpected error writing aws-resource-lab values file")
```

Comment:
I removed the "unexpected error" parts because they where everywhere and did not really transport useful error information. I think all errors are more or less unexpected. It is also that there are occasions where we get error messages ala `unexpected error: unexpected error ...`.